R/add_comp_snfi.R

```diff
@@ -2,32 +2,66 @@
 #'
 #' @param df A data frame.
 #' @param shelter_type_cat Column name for shelter type.
-#' @param shelter_type_cat_levels Levels for shelter type in that order: none, inadequate, adequate, undefined.
+#' @param shelter_type_none Level for no shelter.
+#' @param shelter_type_inadequate Level for inadequate shelter.
+#' @param shelter_type_adequate Level for adequate shelter.
+#' @param shelter_type_undefined Level for undefined shelter.
 #' @param shelter_issue_cat Column name for shelter issue.
-#' @param shelter_issue_cat_levels Levels for shelter issue in that order: none, 7_to_8, 4_to_6, 1_to_3, undefined.
+#' @param shelter_issue_7_to_8 Level for 7 to 8 shelter issues.
+#' @param shelter_issue_4_to_6 Level for 4 to 6 shelter issues.
+#' @param shelter_issue_1_to_3 Level for 1 to 3 shelter issues.
+#' @param shelter_issue_none Level for no shelter issues.
+#' @param shelter_issue_undefined Level for undefined shelter issues.
 #' @param occupancy_cat Column name for occupancy.
-#' @param occupancy_cat_levels Levels for occupancy in that order: high_risk, medium_risk, low_risk, undefined.
+#' @param occupancy_high_risk Level for high risk occupancy.
+#' @param occupancy_medium_risk Level for medium risk occupancy.
+#' @param occupancy_low_risk Level for low risk occupancy.
+#' @param occupancy_undefined Level for undefined occupancy.
 #' @param fds_cannot_cat Column name for fds cannot.
-#' @param fds_cannot_cat_levels Levels for fds cannot. in that order: 4_to_5_tasks, 2_to_3_tasks, 1_task, none, undefined.
+#' @param fds_cannot_4_to_5 Level for 4 to 5 tasks that cannot be done.
+#' @param fds_cannot_2_to_3 Level for 2 to 3 tasks that cannot be done.
+#' @param fds_cannot_1 Level for 1 task that cannot be done.
+#' @param fds_cannot_none Level for no tasks that cannot be done.
+#' @param fds_cannot_undefined Level for undefined fds cannot.
 #'
 #' @export
 add_comp_snfi <- function(
     df,
     shelter_type_cat = "snfi_shelter_type_cat",
```

Comment:
Would propose we stick with name of var_choice

so instead of shelter_type_non --> shelter_type_cat_none

This is done already in add_comp_prot, and maybe in the others --- rationale would apply for all

Reply:
Thank you for reviewing! Suggested updates have been pushed. I also included one more function that I missed originally: add_comp_foodsec.R.

Reply:
Thank you for reviewing! Suggested updates have been pushed. I also included one more function that I missed originally: add_comp_foodsec.R. I will update the tests accordingly.